modules/private-clusters-about-ibm-cloud.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_cloud_public/installing-ibm-cloud-private.adoc
+
+:_content-type: CONCEPT
+[id="private-clusters-about-ibm-cloud_{context}"]
+= Private clusters in IBM Cloud VPC
+
+To create a private cluster on IBM Cloud VPC, you must provide an existing private VPC and subnets to host the cluster. The installation program must also be able to resolve the DNS records that the cluster requires. The installation program configures the Ingress Operator and API server for only internal traffic.
+
+The cluster still requires access to internet to access the IBM Cloud VPC APIs.
+
+The following items are not required or created when you install a private cluster:
+
+* Public subnets
+* Public network load balancers, which support public ingress
+* A public DNS zone that matches the `baseDomain` for the cluster
+
+The installation program does use the `baseDomain` that you specify to create a private DNS zone and the required records for the cluster. The cluster is configured so that the Operators do not create public records for the cluster and all cluster machines are placed in the private subnets that you specify.
+
+[id="private-clusters-limitations-ibm-cloud_{context}"]
+== Limitations
+
+<TBD>
```

Comment:
Let's start with this for limitations (following what Azure did). Will discuss further with Chris and discuss from a user/admin perspective.

"Private clusters on IBM Cloud VPC are subject only to the limitations associated with the existing VPC that was used for cluster deployment."